sklearn/manifold/_locally_linear.py

```diff
@@ -549,15 +549,16 @@ class LocallyLinearEmbedding(TransformerMixin,
                     or matrix type.  This method should be avoided for
                     large problems.
 
-    tol : float, optional
+    tol : float, default=1e-6
         Tolerance for 'arpack' method
         Not used if eigen_solver=='dense'.
 
-    max_iter : integer
+    max_iter : integer, default=100
         maximum number of iterations for the arpack solver.
         Not used if eigen_solver=='dense'.
 
-    method : string ('standard', 'hessian', 'modified' or 'ltsa')
+    method : string ('standard', 'hessian', 'modified' or 'ltsa'),
+             default='standard'
```

Comment:
```suggestion
    method : {'standard', 'hessian', 'modified', 'ltsa'}, default='standard'
```

Reply:
If it does not fit in a single line due to the limitation of the 79 characters, you can break it with

```
    method : {'standard', 'hessian', 'modified', 'ltsa'}, \
            default='standard'
```

Reply:
The backslash is important :)